session.go

```diff
@@ -462,7 +462,11 @@ func (s *session) Execute(sql string) ([]ast.RecordSet, error) {
 		r, err := runStmt(s, st)
 		ph.EndStatement(s.stmtState)
 		if err != nil {
-			log.Warnf("[%d] session error:\n%v\n%s", connID, errors.ErrorStack(err), s)
+			if terror.ErrorEqual(err, kv.ErrKeyExists) {
+				log.Debugf("[%d] session error:\n%v\n%s", connID, errors.ErrorStack(err), s)
```

Comment:
When the error encountered, it's conducive to positioning problems.
I will remove the debug log.